### ISSUE TITLE
Trusty Build Still Using Qt 5.10.1

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#3121]
With PR https://github.com/rstudio/rstudio/pull/3119 most of the official RStudio desktop builds switched to Qt 5.11.1.

There were linking problems on Trusty that prevented use of the pre-built Qt SDK. To get unblocked on Qt 5.11.1 on other platforms, leaving Trusty using 5.10.1.

This issue tracks (potentially) upgrading Trusty to Qt 5.11.1, probably by doing our own custom build of Qt directly on that platform.

**Comment by gtritchie** [comment]
Closing as won't fix; we are no long building for Trusty in 1.3+.
